resolver/resolver.go

```diff
@@ -76,8 +76,9 @@ func (resolver *Resolver) QueryBlocks(ctx context.Context, id interface{}, param
 func (resolver *Resolver) SubmitTx(ctx context.Context, id interface{}, params *jsonrpc.ParamsSubmitTx, req *http.Request) jsonrpc.Response {
 	// Check if the tx is a v1 tx or v0 tx.
 	txVersion := params.Tx.Version
-	if params.Tx.Version == tx.Version0{
-		// We need to always make sure the tx to submit is a v1 tx as
+
+	if params.Tx.Version == tx.Version0 && params.Tx.Selector.IsBurn() {
+		// When burning, we need to always make sure the tx to submit is a v1 tx as
 		// darknode won't accept v0 tx.
 		params.Tx.Version = tx.Version1
 	}
```

Comment:
If we only change the txVersion for burns, would v0 mint be rejected by the darknode. 
I remember darkndoe will reject non-v1 txs now.

Reply:
Nah it only rejects non-V1 burns. V0 mints are still supported.